docs/File-Formats.md

```diff
@@ -1175,12 +1175,12 @@ geneset_def_version: msigdb_6.1
 ### GSVA score data file
 The data file will be a simple tab separated format, similar to the expression data file: each sample is a column, each gene set a row, each cell contains the GSVA score for that sample x gene set combination.
 
-The first column is the GENESET_ID. This contains the EXTERNAL_ID or "stable id" (MsigDB calls this "standard name") of the gene set. The other colums are the sample columns: An additional column for each sample in the dataset using the sample id as the column header.
+The first column is the `geneset_id`. This contains the name or "stable id" (MSigDB calls this "standard name") of the gene set. The other columns are sample columns: An additional column for each sample in the dataset using the sample id as the column header.
```

Comment:
> this contains the name or "stable id"

I guess you mean: this contains the name "stable id"

Reply:
@oplantalech I simplified it.